homeassistant/components/microbees/config_flow.py

```diff
@@ -0,0 +1,87 @@
+"""Config flow for microBees integration."""
+from collections.abc import Mapping
+import logging
+from typing import Any
+
+from homeassistant import config_entries
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers import (
+    aiohttp_client,
+    config_entry_flow as cv,
+    config_entry_oauth2_flow,
+)
+
+from .api import get_api_scopes
+from .const import DOMAIN, VERSION
+from .microbees import MicroBeesConnector
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ConfigFlow(config_entry_oauth2_flow.AbstractOAuth2FlowHandler, domain=DOMAIN):
+    """Handle a config flow for microBees."""
+
+    CONFIG_SCHEMA = cv.config_entry_only_config_schema
+    DOMAIN = DOMAIN
+    VERSION = VERSION
```

Comment:
Please keep the version in here

Reply:
I see it's current set to 0.1.0. In this context, a version should be set to an int. In this case just 1